sae_lens/training/sae_group.py

```diff
@@ -127,7 +142,27 @@ def load_from_pretrained(cls, path: str) -> "SAEGroup" | dict[str, Any]:
 
         # return instance
 
-    def save_model(self, path: str):
+    @classmethod
+    def load_from_pretrained(cls, path: str) -> "SparseAutoencoderDictionary":
+
+        autoencoders = {}
+
+        for sae_name in os.listdir(path):
+            if os.path.isdir(os.path.join(path, sae_name)):
+                autoencoders[sae_name] = SparseAutoencoder.load_from_pretrained(
+                    os.path.join(path, sae_name)
+                )
+
+        # Create the SAEGroup object
+
+        # loaded SAE groups will not contain a cfg that matches the original.
+        # TODO: figure out how to handle this
```

Comment:
I was thinking about this when working on the SAEGroup typing as well. It's tricky since we'd almost want to reconstruct the original config by looking at the values of the sub-configs. But it still feels sort of off somehow. Probably fine for now to just flag for the future.

Reply:
Agreed. Not a big deal for now but a bit weird / wrong.